bitnami/dremio/templates/tls-secret.yaml

```diff
@@ -0,0 +1,50 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and (include "dremio.tls.createSecret" . ) (eq .Values.dremio.tls.autoGenerated.engine "helm") }}
+{{- $ca := genCA "dremio-ca" 365 }}
+{{- $caSecretName := printf "%s-ca-crt" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $caSecretName }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/part-of: dremio
+    {{- /* We need an extra label that applies to both master and non-master coordinators. Taken from upstream dremio chart */}}
+    app: dremio-coordinator
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: kubernetes.io/tls
+data:
+  tls.crt: {{ include "common.secrets.lookup" (dict "secret" $caSecretName "key" "tls.crt" "defaultValue" $ca.Cert "context" $) }}
+  tls.key: {{ include "common.secrets.lookup" (dict "secret" $caSecretName "key" "tls.key" "defaultValue" $ca.Key "context" $) }}
```

Comment:
Should this secret include the `ca.crt`?

Reply:
In others we include it but it's a bit redundant. For consistency reasons I added it.